apps/analytics/src/modules/aggregated_analytics/load_participant_analytics.py

```
@@ -0,0 +1,30 @@
+
+
+def load_participant_analytics(db, timestamp, analytics_type, verbose=False):
+    participant_analytics = db.participantanalytics.find_many(
+        where={"timestamp": timestamp, "type": analytics_type},
+    )
+
+    if verbose:
+        # Print the first participant analytics
+        print(
+            "Found {} analytics for the timespan from {} to {}".format(
+                len(participant_analytics), start_date, end_date
```

Ruff (0.8.0)

Undefined name `start_date`

(F821)